process_report/institute_list.yaml

```diff
@@ -0,0 +1,67 @@
+-   display_name: Northeastern University
+    domains:
+    - northeastern.edu
+-   display_name: Boston University
+    domains:
+    - bu.edu
+-   display_name: Bentley
+    domains:
+    - bentley.edu
+-   display_name: University of Rhode Island
+    domains:
+    - uri.edu
+-   display_name: Red Hat
+    domains:
+    - redhat.com
+-   display_name: Boston Childrens Hospital
+    domains:
+    - childrens.harvard.edu
+-   display_name: McLean Hospital
+    domains:
+    - mclean.harvard.edu
+-   display_name: Massachusetts Eye & Ear
+    domains:
+    - meei.harvard.edu
+-   display_name: Dana-Farber Cancer Institute
+    domains:
+    - dfci.harvard.edu
+-   display_name: Brigham and Women's Hospital
+    domains:
+    - bwh.harvard.edu
+-   display_name: Beth Israel Deaconess Medical Center
+    domains:
+    - bidmc.harvard.edu
+-   display_name: Harvard University
+    domains:
+    - harvard.edu
+    - chemistry.harvard.edu
+-   display_name: Worcester Polytechnic Institute
+    domains:
+    - wpi.edu
+-   display_name: Massachusetts Institute of Technology
+    domains:
+    - mit.edu
+-   display_name: University of Massachusetts Amherst
+    domains:
+    - umass.edu
+-   display_name: University of Massachusetts Lowell
+    domains:
+    - uml.edu
+-   display_name: Code For Boston
+    domains:
+    - codeforboston.org
+-   display_name: Yale University
+    domains:
+    - yale.edu
+-   display_name: Dartmouth College
+    domains:
+    - dartmouth.edu
+-   display_name: Photrek
+    domains:
+    - kenric.nelson@photrek.io
+-   display_name: Positron Networks
+    domains:
+    - srao@positronnetworks.com
```

Comment:
why do you include the entire email here rather than just the domain? It seems this wouldn't work, since you're splitting at the `@` when processing a PI's email.

Reply:
that was my fault in this [PR](https://github.com/CCI-MOC/process_csv_report/pull/90).

Reply:
This was also my fault for not catching it during review. I made a fix in #98